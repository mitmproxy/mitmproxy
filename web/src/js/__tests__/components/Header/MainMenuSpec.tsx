import * as React from "react";
import FlowListMenu from "../../../components/Header/FlowListMenu";
import { render } from "../../test-utils";

test("MainMenu", () => {
    const { asFragment } = render(<FlowListMenu />);
    expect(asFragment()).toMatchSnapshot();
});
