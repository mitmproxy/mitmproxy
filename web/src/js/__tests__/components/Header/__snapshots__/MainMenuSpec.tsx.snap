// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainMenu 1`] = `
<DocumentFragment>
  <div
    class="main-menu"
  >
    <div
      class="menu-group"
    >
      <div
        class="menu-content"
      >
        <div
          class="filter-input input-group"
        >
          <span
            class="input-group-addon"
          >
            <i
              class="fa fa-fw fa-search"
              style="color: black;"
            />
          </span>
          <input
            class="form-control"
            placeholder="Search"
            type="text"
            value="~u /second | ~tcp | ~dns"
          />
        </div>
        <div
          class="filter-input input-group"
        >
          <span
            class="input-group-addon"
          >
            <i
              class="fa fa-fw fa-tag"
              style="color: rgb(0, 0, 0);"
            />
          </span>
          <input
            class="form-control"
            placeholder="Highlight"
            type="text"
            value="~u /path"
          />
        </div>
      </div>
      <div
        class="menu-legend"
      >
        Find
      </div>
    </div>
    <div
      class="menu-group"
    >
      <div
        class="menu-content"
      >
        <div
          class="filter-input input-group"
        >
          <span
            class="input-group-addon"
          >
            <i
              class="fa fa-fw fa-pause"
              style="color: rgb(68, 68, 68);"
            />
          </span>
          <input
            class="form-control"
            placeholder="Intercept"
            type="text"
            value=""
          />
        </div>
        <button
          class="btn-sm btn btn-default"
          title="[a]ccept all"
        >
          <i
            class="fa fa-forward text-success"
          />
          Â Resume All
        </button>
      </div>
      <div
        class="menu-legend"
      >
        Intercept
      </div>
    </div>
  </div>
</DocumentFragment>
`;
